source/runner/Dangerfile.ts

```diff
@@ -1,35 +1,35 @@
 import { DangerResults } from "../dsl/DangerResults"
 import { DangerDSLType } from "../dsl/DangerDSL"
-import { MarkdownString } from "../dsl/Aliases"
+import { MarkdownString, Filename, LineNumber } from "../dsl/Aliases"
 
 export interface DangerContext {
   /**
    * Fails a build, outputting a specific reason for failing
    *
    * @param {MarkdownString} message the String to output
    */
-  fail(message: MarkdownString): void
+  fail(message: MarkdownString, options?: DangerOptions): void
 
   /**
    * Highlights low-priority issues, does not fail the build
    *
    * @param {MarkdownString} message the String to output
    */
-  warn(message: MarkdownString): void
+  warn(message: MarkdownString, options?: DangerOptions): void
 
   /**
    * Puts a message inside the Danger table
    *
    * @param {MarkdownString} message the String to output
    */
-  message(message: MarkdownString): void
+  message(message: MarkdownString, options?: DangerOptions): void
 
   /**
    * Puts a message inside the Danger table
    *
    * @param {MarkdownString} message the String to output
    */
-  markdown(message: MarkdownString): void
+  markdown(message: MarkdownString, options?: DangerOptions): void
```

Comment:
I think so, I can see there are good reasons for wanting to put arbitrary markdown on a file 👍